### PR TITLE
contact us page

### LINKED ISSUE #1: contact us page
As I can see there is no contact us page in the project I want to make that page.
@shriya030  can i work on this issue under LGM-SOC'21

### PR DESCRIPTION
**Pull request by kumarishalini6**
# Description
I have created contact us component 

Fixed #1 

CS Hut - Personal - Microsoft​ Edge 6_10_2021 07_33_01 PM

### CODE REVIEW

**Comment by kumarishalini6** [comment]
@shriya030 please have a look and if you want any changes let me know. Thanks

**Review by aj3011 — CHANGES_REQUESTED** [review]
1. Remove the blur effect but don't remove box-shadow effect
2. Change the background color of the submit button and contact us title to sky blue and text color to black.

**Comment by kumarishalini6** [comment]
> 1. Remove the blur effect but don't remove box-shadow effect
> 2. Change the background color of the submit button and contact us title to sky blue and text color to black.

I didn't add any blur effect , can you please explain where you have seen blur effect?
I 'll make other changes.

**Comment by aj3011** [comment]
Capture

Blur part in box-shadow

**Comment by kumarishalini6** [comment]
> Capture
> 
> Blur part in box-shadow

 isn't blur effect , it is part of box shadow , ok I'll remove that

**Comment by kumarishalini6** [comment]
@shriya030 @aj3011  I have made all the changes please have a look

File: src/App.js [hunk 1]
```diff
@@ -64,16 +65,17 @@ function App() {
...
+        <Route exact path="/frameworks" component={Frameworks} />
+        <Route exact path="/tags" component={Tags} />
+        <Route exact path="/softwares" component={Softwares} />
+        <Route exact path="/development" component={Development} />
+        <Route exact path="/" component={Home}></Route>
+        
+      </Switch>
+      <ScrollToTop />
+
     </div>
```
Comment (shriya030) [inline src/App.js:81]: Expected corresponding JSX closing tag for <> (79:4)

  77 |       <ScrollToTop />
  78 |
> 79 |     </div>
     |     ^
  80 |   );
  81 | }
  82 |

Comment (shriya030) [reply src/App.js:81]: Kindly resolve the errors.

**Review by shriya030 — APPROVED** [review]

**Comment by shriya030** [comment]
@kumarishalini6 Great 🎉

**Comment by kumarishalini6** [comment]
> @kumarishalini6 Great 🎉

Thanks😊

**Review by aj3011 — APPROVED** [review]

**Pull request merged**